django/db/backends/base/features.py

```diff
@@ -74,6 +74,10 @@ class BaseDatabaseFeatures:
     # by returning the type used to store duration field?
     supports_temporal_subtraction = False
 
+    # Does the backend support multiplying and dividing timedeltas by a
+    # scalar value?
+    supports_duration_multiplication = False
```

Comment:
It's possible to multiple and divide `DurationField` by a scalar value on MySQL. I've changed this PR to `Refs` and used  `django_test_skips()` instead of a feature flag.